packages/neuron-wallet/src/services/addresses.ts

```diff
@@ -165,7 +165,8 @@ export default class AddressService {
       AddressPrefix.Mainnet
     ).address
 
-    const blake160: string = LockUtils.addressToBlake160(testnetAddress)
+    const addressToParse = env.testnet ? testnetAddress : mainnetAddress
```

Comment:
Any plan to move this out of `Env`, placing into its own storage/class?

Reply:
Will do this later, don't know how to judge mainnet now.

Reply:
got that.